### PR TITLE
eq-5 error messages template

### PR DESCRIPTION
**Pull request by hamishtaplin**
screen shot 2016-03-23 at 07 50 43
### Changes
- Error messages displayed at top of page inside `.alert` with links to their respective response.
### Test
- `npm run compile`
- Submit form with errors
- Check that output is as per screenshot

### CODE REVIEW

File: tests/app/renderer/test_renderer.py [hunk 1]
```diff
@@ -7,8 +6,7 @@
 class RendererTests(unittest.TestCase):
 
     def test_render(self):
-        renderer = Renderer()
-        self.assertIsInstance(renderer.render(model=None), RenderingContext)
+        pass
```
Comment (warrenbailey) [inline tests/app/renderer/test_renderer.py]: You've just removed the unit test, we really should replace this with a valid test.

**Comment by warrenbailey** [comment]
Missing unit tests for the renderer :-1:

**Comment by warrenbailey** [comment]
Needs rebasing too

File: tests/app/renderer/test_renderer.py (hunk 1 above)
Comment (hamishtaplin) [inline tests/app/renderer/test_renderer.py]: I blame @weapdiv-david :-1:

**Comment by warrenbailey** [comment]
:+1:

**Pull request merged**